pkg/operator/ceph/pool/controller.go

```diff
@@ -295,6 +295,7 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 				logger.Debug("external rgw endpoint monitoring go routine already running!")
 			} else {
 				checker := newMirrorChecker(r.context, r.client, r.clusterInfo, request.NamespacedName, &cephBlockPool.Spec, cephBlockPool.Name)
+				r.blockPoolChannels[cephBlockPool.Name].monitoringRunning = true
```

Comment:
I just re-ran the CI, I suspect this is causing an issue with the CI, not sure yet.